packages/client-mcp/README.md

```
@@ -0,0 +1,280 @@
+<!-- GENERATED by `./builder docs:export` — DO NOT EDIT. Source: docs/mcp/readme.md -->
+
+<p align="center">
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)